### DISCUSSION TITLE
Request for Group by souce

### DISCUSSION THREAD

**Discussion by chujiamo** [iptv-org/community#1146] [Ideas]
Can the m3u sortted by souces? e.g. bbc?

**Comment by freearhey** [comment]
You can just use playlists from the repository directly by clicking on the “Raw” button in the top right corner:

![image](https://github.com/user-attachments/assets/b918ef97-99e2-4990-9c82-63639eb494e7)

https://github.com/iptv-org/iptv/blob/master/streams/uk_bbc.m3u
